runtime/lua/vim/treesitter/query.lua

```diff
@@ -0,0 +1,176 @@
+local a = vim.api
```

Comment:
Decent amount of spots in this function missing spaces between arguments to functions.